server/api.js

```diff
@@ -1,59 +1,39 @@
 "use strict"
 
+const denodeify = require("denodeify")
```

Comment:
Does this work? In the past i haven't been able to use denodeify because it was screwing up the prototypical ownership.